### ISSUE TITLE
Empty document results in infinite import job

### ISSUE THREAD

**Issue by t0phr** [N-Schaef/shreddr#86]
Worse, after a restart of the server, shreddr seems to continue where it stopped, and no further documents are processed.

Some log output:
```
shreddr    | 16:20:53 [ERROR] OCR attempt did not yield text.                                                                                                                                                                                                                    
shreddr    | 16:20:54 [WARN] Could not tag document 10: body of document 10 is empty                                              
shreddr    | thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: DocRepoError(LocalRepoError(DocumentMissingBody(10)))', src/main.rs:150:31
shreddr    | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
Obligatory stop-the-shreddr-document: [doc_000154.pdf](https://github.com/N-Schaef/shreddr/files/6290276/doc_000154.pdf)
Shreddr version: v0.2.0

**Comment by t0phr** [comment]
Deleting the problematic files from the `consume` directory is a workaround

**Comment by N-Schaef** [comment]
Fixed in 8e0b3c5 

Now the empty body error is just a warning.

**Comment by N-Schaef** [comment]
Import of new files was still broken, as the hash was computed on the modified (ocr+optimization) files.
This was now finally fixed in b8af38d
